wordpress-mobile/WordPress-FluxC-Android
# Use Plugin DSL

### Context

This PR is about migrating Gradle plugin's configuration to the Plugin DSL. Please see #1960 for more details.

### How to test

1. Checkout `develop`
2. Run `./gradlew help`
3. Make sure there's `The Kotlin Gradle plugin was loaded multiple times in different subprojects (...)` warning printed
4. Checkout `issue/1960-use-plugin-dsl`
5. Run `./gradlew help`
6. Make sure there is no warning from point 3.


Closes: #1960
